include-fragment-element.js

```diff
@@ -57,9 +57,14 @@
   });
 
   IncludeFragmentPrototype.attributeChangedCallback = function(attrName) {
-    // Reload data load cache
     if (attrName === 'src') {
-      getData(this);
+      // Reload data load cache.
+      var data = getData(this);
+
+      // Source changed after attached so replace element.
+      if (this.parentNode) {
```

Comment:
You'd want to do a full `document` contains check here. The parent might be detached.

Though, it would be easier just to record the current attached state from the callback.

``` js
.attachedCallback = function() {
  this._attached = true
}
```

Then `if (this._attached)` here.